yeaseul/src/components/Nav.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+const items = [
+  { src: "korea.png", number: 0 },
+  { src: "japan.png", number: 1 },
+  { src: "china.png", number: 2 },
+  { src: "USA.png", number: 3 },
+];
+
+const Nav = ({ setter }: {setter: React.Dispatch<React.SetStateAction<number>>}) => {
```

Comment:
이렇게 인라인 타입 보다는 인터페이스 사용을 권장해요 ~
```suggestion
interface Props {
 setter: Dispatch<SetStateAction<number>>;
}

const Nav = ({ setter }: Props) => {
```

Reply:
😮오..넵!